2-mandatory/2-financial-times.js

```diff
@@ -14,24 +20,58 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    
+    let shortestLine;
+    let length_headline = 100000;
+
+    for (headline of allArticleTitles) {
+        let currentTitleWordCount = headline.split (" ").length;
+        
+     if (currentTitleWordCount < length_headline) {
+        shortestLine = headline;
+        length_headline = currentTitleWordCount;
+     }
+    }
+        
+    return shortestLine;
+    
 }
+    
+
 
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
+
```

Comment:
Very nice solution!
I like that you've used a separate function to check if the string has a number. It makes the code very easy to read 👍